base_utils.py

```
@@ -263,5 +263,6 @@ async def process_with_overlap(req, attribute_service, POST_PROMPT, Posts):
             )
             all_dict_meta+= result['post_meta']
         except Exception as e:
-            print(f'❌ Lỗi xử lý batch: {batch}\n{e}')
```

Removed helpful error logging: the original print included the batch and exception details. Silently replacing that with an unrelated debug message makes debugging failures difficult. Use proper logging (logger.exception or logger.error) to record exception details instead of bare prints.